src/main/scala/strawman/collection/View.scala

```diff
@@ -33,6 +33,7 @@ object View extends IterableFactory[View] {
   /** Avoid copying if source collection is already a view. */
   def fromIterable[E](it: Iterable[E]): View[E] = it match {
     case it: View[E] => it
+    case it: Iterable[E] => it.view
     case _ => View.fromIterator(it.iterator())
   }
```

Comment:
The wildcard pattern is unreachable.

I think we should just implement it as `it.view` (without doing pattern matching).

Reply:
Yes, that sounds better, I'll change that.